packages/component-library/components/Spacing/types.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Spacing is in grid units (24px grid)
+ */
+export type Spacing = {
+  p?: ResponsiveSpacing
+  pt?: GridFractions
+  pr?: GridFractions
+  pb?: GridFractions
+  pl?: GridFractions
+  px?: GridFractions
+  py?: GridFractions
+  m?: ResponsiveSpacing
+  mt?: GridFractions
+  mr?: GridFractions
+  mb?: GridFractions
+  ml?: GridFractions
+  mx?: GridFractions
+  my?: GridFractions
+}
+
+export type GridFractions =
+  | 0
+  | 0.25
+  | 0.5
+  | 0.75
+  | 1
+  | 1.25
+  | 1.5
+  | 1.75
+  | 2
+  | 2.5
+  | 3
+  | 3.5
+  | 4
```

Comment:
I think we need to come up with a solid solution to this before we ship it. @ActuallyACat I vaguely remember you saying that we might confine this list to the Headings only, rather than having them available for other components – is that right?

Reply:
These have been moved from the "global types" (`packages/component-library/types.ts`) to be within the Spacing package. At this time, the spacing classes only support these units so I think it's fairly safe to keep it here.

Reply:
Ah right. I assumed that the fact that Spacing was its own package meant that it was intended to be used with multiple components